### DISCUSSION TITLE
Menu Items Duplicated

### DISCUSSION THREAD

**Discussion by GlenWarren** [orchidsoftware/platform#2462] [Q&A]
We are experiencing a very strange 'bug'.

When you navigate to a new page via a Menu item on the left, all of the Menu items are duplicated in the menu bar. And they continue to duplicate, each time you navigate to a new page. The items in the Profile menu are also duplicated. See example:

<img width="291" alt="Screenshot 2022-12-02 at 15 20 56" src="https://user-images.githubusercontent.com/44340845/205326229-c71acba8-1cbe-4777-b5c3-b657470d5f25.png">

What's even stranger, is that this is only happening for us in our production environment, and not locally.

Would appreciate some help.

**Comment by tabuna** [comment]
- What is your production environment? 
- What package version are you using? 
- Do you use Laravel Octane?

**Reply by GlenWarren** [reply]
- Not sure how to answer your first question.
- We are using "orchid/platform": "12.2.1".
- We are not using Laravel Octane.

**Reply by tabuna** [reply]
Based on this and a previous question on the forum. It seems that you have some sort of intermediate caching layer. Which reproduces the very first request, look that way.
